auto_submit/bin/server.dart

```diff
@@ -5,11 +5,13 @@
 import 'dart:async';
 
 import 'package:auto_submit/helpers.dart';
-import 'package:shelf_router/shelf_router.dart';
 import 'package:auto_submit/requests/github_webhook.dart';
+import 'package:auto_submit/service/config.dart';
```

Comment:
This is great progress!

Can you split this PR into several smaller ones. In it's current state, its hard to give feedback.

I think something like:
1. Github app authentication
2. GraphQL query
3. Have the webhook work just with an approval, and merge it
4. Later we can follow up with all the extra logic

This will ensure we have high quality tests for your project

Reply:
Sure Casey, l will try to split them.